### ISSUE TITLE
Codecheg MAXDIFF problem in C

### ISSUE THREAD

**Issue by ritish099** [OpenGenus/cosmos#4824]
**This is a(n):**

- [x] New algorithm
- [ ] Update to an existing algorithm
- [ ] Error
- [x] Proposal to the Repository

**Details:**

Chef has gone shopping with his 5-year old son. They have bought N items so far. The items are numbered from 1 to N, and the item i weighs Wi grams.

Chef's son insists on helping his father in carrying the items. He wants his dad to give him a few items. Chef does not want to burden his son. But he won't stop bothering him unless he is given a few items to carry. So Chef decides to give him some items. Obviously, Chef wants to give the kid less weight to carry.

However, his son is a smart kid. To avoid being given the bare minimum weight to carry, he suggests that the items are split into two groups, and one group contains exactly K items. Then Chef will carry the heavier group, and his son will carry the other group.

Help the Chef in deciding which items should the son take. Your task will be simple. Tell the Chef the maximum possible difference between the weight carried by him and the weight carried by the kid.

**Comment by wokira** [comment]
wokira:add_maxdiff #4836

**Comment by ritish099** [comment]
@wokira i have submitted the PR first in C try to submit in other language

**Comment by wokira** [comment]
I am sorry,the title said solution in C that's why I submitted in C. 
I can do it in java and python - should I proceed?

**Comment by ritish099** [comment]
Yeah ofcourse you can procced in any language of your choice . As i am not a mentor but i suggested you from my past observations on cosmos. For more info pls contact @arnavb

**Comment by wokira** [comment]
@ritish099  Thankyou for the advice.  @arnavb I can work on this for java and python. Please assign me this issue if I am allowed to proceed. Thankyou.

**Comment by wokira** [comment]
Java and Python Codes for MAXDIFF #4836
